common/event/emitter.js

```diff
@@ -1,51 +1,71 @@
 /**
```

Comment:
It has `once`: https://github.com/zerious/cedar/blob/master/common/event/emitter.js

Also, Node's event emitters don't specifically support streams. It's the other way around. A stream **is** an EventEmitter. There's no occurrence of "stream" here: https://github.com/joyent/node/blob/master/lib/events.js